### ISSUE TITLE
Print newline after progress bar before printing warning

### ISSUE THREAD

**Issue by CameronBieganek** [alan-turing-institute/MLJ.jl#556]
This isn't a big deal, but it would be nice if these warning messages would print on a new line instead of being attached to the progress bar:

![image](https://user-images.githubusercontent.com/8310743/82952922-76b54980-9f6f-11ea-8f2d-885a963cdb08.png)

**Comment by OkonSamuel** [comment]
Adding a new line after each ProgressMeter printing solves this issue but would lead to other issues
```julia
using ProgressMeter
p = Progress(10)
for i in 1:10
           sleep(2*rand())
           next!(p);print("\n");#@warn "yes"
       end
```
Progress:  10%|████▏                                    |  ETA: 0:01:08
Progress:  20%|████████▎                                |  ETA: 0:00:37
Progress:  30%|████████████▎                            |  ETA: 0:00:22
Progress:  40%|████████████████▍                        |  ETA: 0:00:15
Progress:  50%|████████████████████▌                    |  ETA: 0:00:11
Progress:  60%|████████████████████████▋                |  ETA: 0:00:08
Progress:  70%|████████████████████████████▊            |  ETA: 0:00:06
Progress:  80%|████████████████████████████████▊        |  ETA: 0:00:04
Progress:  90%|████████████████████████████████████▉    |  ETA: 0:00:02
Progress: 100%|█████████████████████████████████████████| Time: 0:00:15

Another solution is to tell package owners to print a newline before issuing a warning (I doubt they would like that) 
Do you have any other ideas?

**Comment by ablaom** [comment]
Yeah, it looks like the package in this case (MLJLinearModels?) does not use the "sanctioned" `@warn` macro for logging the warning.

**Comment by ablaom** [comment]
Oh, yes is does.
